terraform/deployments/govuk-publishing-infrastructure/clamav_efs.tf

```diff
@@ -0,0 +1,28 @@
+locals {
+  clamav_efs_name = "clamav-efs-${local.cluster_name}"
+}
+
+resource "aws_efs_file_system" "clamav-efs" {
```

Comment:
Consider putting a `Name` and/or `Description` tag on this to make it clear that it's for the ClamAV *signatures* (database). Or maybe just work it into the name somehow? (Could probably lose the `efs` from the name in most contexts, but signposting that it's for the sigs as opposed to, say, the files being scanned or anything like that, is quite helpful.)

Reply:
`clamav-sigs-db`? `clamav-db`? 🤷

Reply:
I will, thanks